docs/content/guides/rows/row-height.md

```diff
@@ -16,21 +16,23 @@ tags:
   - row dimmensions
   - manual resize
 react:
-  metaTitle: Row height - React Data Grid | Handsontable
+  metaTitle: Row heights - React Data Grid | Handsontable
 searchCategory: Guides
 ---
 
-# Row height
+# Row heights
+
+Configure row heights, using an array or a function. Let your users manually change row heights using Handsontable's interface.
```

Comment:
> Configure row heights, using an array or a function.

It's more like

> Configure row heights, using a number, an array, or a function.

because for `colWidths` and `rowHeights` we can also use a single number, and then the values are set for all the rows/columns.